src/app/Plans/PlansPage.tsx

```diff
@@ -10,7 +10,7 @@ import {
   Button,
   Alert,
 } from '@patternfly/react-core';
-import { Link } from 'react-router-dom';
+import { Link, useHistory } from 'react-router-dom';
```

Comment:
This Link import is unused now, let's remove it

Reply:
Of course!